commit 79ca825d3bea16b06a9684fba09e69e91828be1b (refs/changes/85/1585/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-02-27T17:44:40-08:00 (2 years, 7 months ago)
    
    joyent/node-cueball#87 documentation refers to DNSResolver, but this is not exported in index.js

diff --git a/CHANGES.adoc b/CHANGES.adoc
index d92829d..f24f900 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,15 @@ toc::[]
 
 ## v2.x
 
+### v2.2.0
+
+New minor release, due to addition of new API.
+
+API changes:
+
+ - The `DNSResolver` class is now exported in the package root, as specified
+   in the documentation. It was mistakenly left out.
+
 ### v2.1.1
 
 Maintenance release.
diff --git a/lib/index.js b/lib/index.js
index dcbdd57..28a9b1c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -20,6 +20,7 @@ module.exports = {
 	ConnectionPool: mod_pool.ConnectionPool,
 	ConnectionSet: mod_cset.ConnectionSet,
 	Resolver: mod_resolver.Resolver,
+	DNSResolver: mod_resolver.DNSResolver,
 	StaticIpResolver: mod_resolver.StaticIpResolver,
 	resolverForIpOrDomain: mod_resolver.resolverForIpOrDomain,
 
diff --git a/package.json b/package.json
index d526295..d5523c4 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "2.1.1",
+  "version": "2.2.0",
   "description": "manage a pool of connections to a multi-node service where nodes are listed in DNS",
   "main": "lib/index.js",
   "dependencies": {
